_posts/patterns/2015-04-15-icons.md

```diff
@@ -137,7 +137,7 @@ info:               Icons are used to clearly and visually represent some inform
             </span>
         </div>
         <div class="icon-classname">
-            Enter fullscreen
+            arrows-alt (Enter Fullscreen)
```

Comment:
Ah, I renamed it to just "arrows" in the font definition. I think I'm going to change it to "fullscreen" in both.